packages/utils/src/getDOMInfo.ts

```diff
@@ -55,6 +55,9 @@ export const styleInFlow = (el: HTMLElement, parent: HTMLElement) => {
 
   if (style.overflow && style.overflow !== 'visible') return;
   if (parentStyle.float !== 'none') return;
+  if (parent && parentStyle.display === 'grid') {
```

Comment:
Trying to understand the mechanism under the hood for this logic. Why does `flex` specifically check for `flex-direction: column`? 🤔

Reply:
One of the oldest part of Craft 😄 So this part is related to the placement logic and it's mostly borrowed from GrapesJS. 

Let's say we have 2 elements - `A` for the element we're dragging and `B` for the element we're hovering over.

If element B is in "flow" - then we would only consider B's y-axis threshold relative to the current cursor position to determine if A should be dropped before or after B. This is because some elements are arranged vertically, and so naturally when you're gonna drop something at these vertically arranged elements, you would move the cursor from top-to-bottom rather than from left-to-right.

For example, if a child element's parent is of `flex-direction: column`, it would mean that it takes up the entire width of its parent, and should be arranged vertically. 
> Related to: https://github.com/artf/grapesjs//issues/1626

Reply:
Ah, so it would be different in the case of `grid` – even if it's `grid-auto-flow: row` we can't assume elements are all arranged vertically?

Reply:
Yea, we would probably require some additional heuristics

Reply:
@timc1 Shall we merge this one in as a temporary solution for grid layouts? Then, we can revisit this issue with a more complete solution later?

Reply:
@prevwong let's do that 👍